src/elements/content-sidebar/activity-feed/annotations/messages.js

```diff
@@ -0,0 +1,31 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+    annotationActivityDeleteMenuItem: {
+        id: 'be.contentSidebar.activityFeed.annotationActivity.annotationActivityDeleteMenuItem',
+        defaultMessage: 'Delete',
+        description: 'Text to show on menu item to delete the annotation',
+    },
+    annotationActivityDeletePrompt: {
+        id: 'be.contentSidebar.activityFeed.annotationActivity.annotationActivityDeletePrompt',
+        defaultMessage: 'Are you sure you want to permanently delete this comment?',
```

Comment:
Would this be deleting the comment or the annotation?

Reply:
I would think it would be deleting the comment but ya know that is a good question. I'll follow up about it.